python/cuml/benchmark/run_benchmarks.py

```diff
@@ -142,13 +146,35 @@ def extract_param_overrides(params_to_sweep):
         action='store_true',
         help='Throw exception on a failed benchmark',
     )
+    parser.add_argument(
+        '--algorithms',
+        dest='print_algorithms',
+        action='store_true',
+        help='Print the list of all available algorithms and exit',
+    )
+    parser.add_argument(
+        '--datasets',
```

Comment:
Same. If you call this `print-datasets` you won't even have to specify a dest anymore.